### PR TITLE
Fixes #22571 - allocation field i18n idempotent

### PR DESCRIPTION
**Pull request by lizagilman**
This bug only happens when using foreman in any language other then English.
Bug description at redmine.

### CODE REVIEW

**Comment by lizagilman** [comment]
[test foreman]
[test katello]

**Comment by lizagilman** [comment]
[test foreman]
[test katello]

**Comment by lzap** [comment]
@lizagilman I thought this was working and we could not reproduce, what exactly are you trying to do here please?

File: app/helpers/hosts_helper.rb [hunk 1]
```diff
@@ -407,7 +407,8 @@ def allocation_text_f(f)
           :name => 'allocation_radio_btn',
           :class => (label == active) ? 'btn btn-default active' : 'btn btn-default',
           :onclick => "tfm.computeResource.libvirt.allocationSwitcher(this, '#{label}');",
-          :data => { :toggle => 'button' }
+          :data => { :toggle => 'button' },
+          :id => (label == 'Full') ? 'btnFull' : nil
```
Comment (lzap) [inline app/helpers/hosts_helper.rb]: I'd rather do less generic id like "btnAllocationFull". Also, `nil` is not a good id at all, we aim to have ids for all elements for the future, can you set something like "btnAllocationNonFull"?

**Review by lzap — CHANGES_REQUESTED** [review]
I will test for regressions, I am curious what this fixes.

File: app/helpers/hosts_helper.rb (hunk 1 above)
Comment (lizagilman) [reply app/helpers/hosts_helper.rb]: should we have ids even if there is no later use of them?

**Comment by lizagilman** [comment]
@lzap updated + comment inside + added description

**Review by lzap — APPROVED** [review]

File: app/helpers/hosts_helper.rb (hunk 1 above)
Comment (lzap) [reply app/helpers/hosts_helper.rb]: It's for QA so they can automate web browser bots. Anyway, I am merging, thanks.

**Comment by lzap** [comment]
For the record, this commit broke tests, but it is not related (confirmed by @dLobatog) it has something to do with new dependency.

**Pull request merged**